testsuite/ui/views/admin/settings/tokens.py

```diff
@@ -48,7 +48,9 @@ class TokenNewView(BaseSettingsView):
 
     path_pattern = "/p/admin/user/access_tokens/new"
     name = TextInput(id="access_token_name")
-    scopes = CheckBoxGroup("//*[@id='access_token_scopes_input']")
+    scopes = PfCheckBoxGroup(
+        '//*[@id="new_access_token"]/div[@class="pf-c-form__group"][.//span[contains(normalize-space(.), "Scopes")]]'
```

Comment:
Maybe simplified selector`"//*[@for='access_token_scopes']/ancestor::div[@class='pf-c-form__group']"`